src/main/java/com/dev/theater/model/dto/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.theater.model.dto;
+
+public class MovieSessionRequestDto {
+    private String movieTitle;
+    private String showTime;
+    private Long cinemaHallId;
+    private String cinemaHallDescription;
```

Comment:
the only informative fields that we can get from this dto are:
`movieId`
`cinemaHallId`
`showTime`

let's refactor the fields